main.py

```diff
@@ -45,10 +44,14 @@ def get_stats() -> str:
         lang_data = data['data']['languages']
     except KeyError:
         print("Please Add your WakaTime API Key to the Repository Secrets")
-        sys.exit(1)
+        
 
     data_list = []
-    pad = len(max([l['name'] for l in lang_data[:5]], key=len))
+    try:
+        pad = len(max([l['name'] for l in lang_data[:5]], key=len))
+    except ValueError:
+        print("The Data seems to be empty. Please wait for a day for the data to be filled in.")
```

Comment:
https://github.com/athul/waka-readme/issues/18#issuecomment-662504219 Please check this comment, i think it is better, Just add a message to the readme. What do you think?

Reply:
Message in the Readme is a bit too much I'd say. We don't want people visiting the profile get a bad impression that the data is missing.

Reply:
Or on 2nd thoughts, that will be better than to silently failing

Reply:
Then it's okay. Show message on the console and silent exit.